app/uk/gov/hmrc/merchandiseinbaggage/controllers/AgentDetailsController.scala

```diff
@@ -40,14 +40,14 @@ class AgentDetailsController @Inject()(
   val onPageLoad: Action[AnyContent] = actionProvider.journeyAction { implicit request =>
     val preparedForm = request.declarationJourney.maybeCustomsAgentName.fold(form)(form.fill)
 
-    Ok(view(preparedForm, backButtonUrl))
+    Ok(view(preparedForm, backButtonUrl, request.declarationType))
```

Comment:
`implicit request` is already available in the Layout.view , so we don't have to pass the `declarationType` explicitly to each view i think (?), but i see its already merged, not a big deal